packages/website/components/account/storageManager/storageManager.js

```diff
@@ -85,6 +86,11 @@ const StorageManager = ({ className = '', content }) => {
 
   return (
     <div ref={storageManagerRef} className={clsx('section storage-manager-container', className)}>
+      <Link href={'account/payment'} passHref>
+        <a href={'account/payment'} className="storage-manager-payment-link">
```

Comment:
I thought you only needed href on the outer link

Reply:
it works fine without the second href, but causes a linter error.  let's leave this as-is for now.